docs/examples_benchmark.md

```diff
@@ -0,0 +1,68 @@
+# Real-world Example Benchmarks
+For all three real-word examples (PNEUMONIA, NER, CCFRAUD), we perform benchmark testing to assess three main aspects: the training overhead, model performance, and scalability of training under FL setting. The main purpose of this benchmark is to show that FL has been implemented correctly and works as expected, rather than to focus on the exact metric value shown below, which will vary based on distinct models and datasets. In the following experiments, each silo is trained with the same hyperparameters to ensure the consistency of results, and the model performance was assessed with all test data combined from silos. Below are the results for each aspect.
+
+## Table of contents
+
+- [Training Overhead](#real-world-example-benchmarks)
+- [Model Performance](#model-performance)
+- [Scalability with Training](#scalability-with-training)
+
+## Training Overhead
+
+For training overhead, there are two main questions of interest: 1. What is the extra wall-clock time spent on training with FL, compared to train a regular centralized model, but only with 1/#silo of the data. 2. What is the extra computing time (summed from all silos) spent on training with FL, compared to train a regular centralized model with data from all silos combined. The first point is important as it indicates how quickly customers can get their model results, and the second point is essential as it is an indication of the money that customers will spend on computing resource. For this reason, we tested 3 models: "FL" for model trained with FL in 3 silos, "Centralized-1/3" for 1 model with 1/3 data, and "Centralized-1" for 1 model with all data combined. 
```

Comment:
use numbering here to provide some visual space, so that the page is easier to read:

1. ... (return)
2. ... (return)

For overhead analysis, I would also explain more precisely what each metric means: what is "wall clock" and what is "compute time". This might sound very basic, but it will help clarify doubts for the reader.